### PR TITLE
bugfix: slow aes encryption

### PR DESCRIPTION
**Pull request by ndr-brt**
Supersedes #721

### CODE REVIEW

File: edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -7,16 +7,14 @@
 #
 #  SPDX-License-Identifier: Apache-2.0
 #
-#  Contributors:
-#       Mercedes-Benz Tech Innovation GmbH - Initial Dockerfile
-#
```
Comment (florianrusch-zf) [inline edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile:12]: 1. Why have you remove the contributors? Mistake?
2. Should we update the contributors list with our two companies?

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

File: edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile (hunk 1 above)
Comment (ndr-brt) [reply edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile:12]: I guess it's a rebase error

**Comment by ndr-brt** [comment]
@florianrusch-zf rebased against `hotfix-0.1.5`, contributors should be ok

File: docs/release-notes/Version 0.1.5.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
 # Release Notes Version 0.1.5
 
+13.02.2023
+
 ## 0. Summary
 
 1. Version updates:
```
Comment (florianrusch-zf) [inline docs/release-notes/Version 0.1.5.md:7]:
```suggestion
1. [Version updates](#1-version-updates)
```

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**